src/pages/Account.vue

```diff
@@ -7,38 +7,43 @@
 
                 <transition name="fade"><div v-if="errorMessage" v-bind:class="[goodFeedback ? 'good message' : 'message' ]">{{ errorMessage }}</div></transition>
 
+                <div v-if="voucherPayments.length > 0">
```

Comment:
Just tested with Steph and Alex - It's definitely returning an array with the array we want to check the length of inside of it.